.github/workflows/tests.yml

```diff
@@ -26,16 +30,19 @@ jobs:
         - name: Run PHPStan
           run: vendor/bin/php-cs-fixer fix --dry-run --diff
 
-
     phpstan:
         runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['8.1', '8.2']
```

Comment:
Same here, static analysis doesn't require to be execute on multiple PHP versions.